ci/base.Dockerfile

```
@@ -53,5 +54,5 @@ RUN pyenv update && \
 
 ENV PATH="/root/.pyenv/shims:${PATH}"
 
-
-RUN pip install --upgrade pip setuptools wheel tox cupy-cuda11x==12.3.0
+ARG CUPY_PACKAGE=cupy-cuda12x
+RUN pip install --upgrade pip setuptools wheel tox ${CUPY_PACKAGE}==12.3.0
```

```suggestion
RUN pip install --upgrade pip setuptools wheel tox
```
Can you try if this works? We install it also in tox by specifying `SUBVARIANT: [-cuda11x]`

---

It doesn't work, some new issues appear. I suggest to do this change as a separate PR.